packages/ui-components/src/globals/globals.ts

```diff
@@ -23,7 +23,9 @@ export const initialize = (userConfig: UIComponentsConfig = {}) => {
 					console.warn(`Invalid kv-component mode: ${elmMode}, expected: ${Object.values(StyleMode).join(',')}`);
 				}
 			}
-			elm = elm.parentElement;
+			// TODO: check with more time why cant obtain the parentElement
+			// elm = elm.parentElement;
+			elm = doc.body;
```

Comment:
Was this intended?

Reply:
This was a fix requested by @ricardogoncalves89 to address a bug. It's a temporary fix.